CHANGELOG.md

```diff
@@ -145,6 +145,22 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 
 ### Linter
 
+#### BREAKING CHANGES
+
+- [useNamingConvention](https://biomejs.dev/linter/rules/use-naming-convention) and [useFilenamingConvention](https://biomejs.dev/linter/rules/use-filenaming-convention) now require identifiers to be in ASCII and without consecutive delimiters.
+
+  Set the `requireAscii` rule option to `false` to allow non-ASCII identifiers.
```

Comment:
Breaking changes are always bad. Hence, I would suggest having a code block that shows the whole option, users can copy-paste it and don't need to think twice